_config.yml

```diff
@@ -9,10 +9,10 @@ playstore_link                            :
 presskit_download_link                    : #https://emilbaehr.com                                          # Enter a link to downloadable file or (e.g. public Dropbox link to a .zip file). 
                                                                                       # Or upload your press kit file to assets and set path accordingly (e.g. "assets/your_press_kit.zip").
 
-app_icon                                  : assets/appicon.png                      # Automatically populates if not set and if iOS app ID is set.  Otherwise enter path to icon file manually.
+app_icon                                  :                                           # Automatically populates if not set and if iOS app ID is set.  Otherwise enter path to icon file manually.
```

Comment:
why did you delete this ?

Reply:
According to the comment the app icon is is automatically pulled from the iOS app ID (provided), this overused that and gives the icons present in assets... which doesn't seem to be the correct icon for the app.

Reply:
Any update?